apps/netting/makefile

```diff
@@ -0,0 +1,7 @@
+all: clean gen_data
+
+clean:
+	rm -rf data/
+
+gen_data:
+	python3 scripts/generate_data.py
```

Comment:
Just to remain consistent with the code base, let's use `Makefile` instead of `makefile`. Although both work, the [GNU `make`](https://www.gnu.org/software/make/manual/make.html#Makefile-Names) manual recommends `Makefile`:

> We recommend `Makefile` because it appears prominently near the beginning of a directory listing, right near other important files such as `README`.